trunk/SUAVE/Components/Energy/Networks/Vectored_Thrust.py

```diff
@@ -0,0 +1,326 @@
+        conditions.propulsion.pitch_command = self.pitch_command
+        
+        # Run the propeller     
+        F, Q, P, Cp , output, etap = propeller.spin_variable_pitch(conditions)
+            
+        # Check to see if magic thrust is needed, the ESC caps throttle at 1.1 already
+        eta        = conditions.propulsion.throttle[:,0,None]
+        P[eta>1.0] = P[eta>1.0]*eta[eta>1.0]
+        F[eta>1.0] = F[eta>1.0]*eta[eta>1.0]
+
+        # Run the avionics
+        avionics.power()
+
+        # Run the payload
+        payload.power()
+        
+        # Run the motor for current
+        i, etam = motor.current(conditions)  
+        
+        # Fix the current for the throttle cap
+        motor.outputs.current[eta>1.0] = motor.outputs.current[eta>1.0]*eta[eta>1.0]
+         
+        # link
+        esc.inputs.currentout =  motor.outputs.current
+        
+        # Run the esc
+        esc.currentin(conditions)
+
+        # Calculate avionics and payload power
+        avionics_payload_power = avionics.outputs.power + payload.outputs.power
+
+        # Calculate avionics and payload current
+        avionics_payload_current = avionics_payload_power/self.voltage
+
+        # link
+        propeller_current       = esc.outputs.currentin*num_engines
+        total_current           = propeller_current + avionics_payload_current
+        battery.inputs.current  = total_current
+        
+        battery.inputs.power_in = -(esc.outputs.voltageout*esc.outputs.currentin*num_engines + avionics_payload_power)
+        battery.energy_calc(numerics)        
+        
+        # Pack the conditions for outputs
+        rpm                  = motor.outputs.omega*60./(2.*np.pi)
+        a                    = conditions.freestream.speed_of_sound
+        R                    = propeller.tip_radius      
+        
+        rpm                  = motor.outputs.omega*60./(2.*np.pi)
+        current              = esc.outputs.currentin
+        battery_draw         = battery.inputs.power_in 
+        battery_energy       = battery.current_energy
+        voltage_open_circuit = battery.voltage_open_circuit
+        voltage_under_load   = battery.voltage_under_load    
+          
+        conditions.propulsion.rpm                             = rpm
+        conditions.propulsion.current                         = current
+        conditions.propulsion.battery_draw                    = battery_draw
+        conditions.propulsion.battery_energy                  = battery_energy 
+        conditions.propulsion.voltage_open_circuit            = voltage_open_circuit
+        conditions.propulsion.voltage_under_load              = voltage_under_load  
+        conditions.propulsion.motor_torque                    = motor.outputs.torque
+        conditions.propulsion.propeller_torque                = Q
+        conditions.propulsion.motor_efficiency                = etam
+        conditions.propulsion.acoustic_outputs[propeller.tag] = output
```

Comment:
Units again.

Reply:
Standard Units used

Reply:
I think you bring it a good point here. It is not. I will change it to Wh/kg and change all similar files